src/main/java/io/reactivex/Scheduler.java

```diff
@@ -292,12 +392,20 @@ public Disposable schedule(@NonNull Runnable run) {
         public abstract Disposable schedule(@NonNull Runnable run, long delay, @NonNull TimeUnit unit);
 
         /**
-         * Schedules a cancelable action to be executed periodically. This default implementation schedules
-         * recursively and waits for actions to complete (instead of potentially executing long-running actions
-         * concurrently). Each scheduler that can do periodic scheduling in a better way should override this.
+         * Schedules a cancelable action to be executed periodically.
```

Comment:
Maybe remove "cancelable" or add it to other "scheduleXXX" methods? 

Other "scheduleXXX" methods also allow cancellation, but their javadoc only mentions it in the description of returned `Disposable`